src/utils.ts

```diff
@@ -1,5 +1,7 @@
 import isPlainObject from 'lodash/isPlainObject';
 
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
 export default function objToKey(obj: Record<string, any>): null | string {
```

Comment:
I think `objToKey` shouldn't be a default export here.
```suggestion
 export function objToKey(obj: Record<string, any>): null | string {
```